include/dao/db_manager.js

```diff
@@ -152,19 +152,33 @@ module.exports = function DBManagerModule(pb) {
          * defer the reporting of an error until the end.
          */
         this.processIndices = function(procedures, cb) {
+            var self = this;
             if (!util.isArray(procedures)) {
                 cb(new Error('The procedures parameter must be an array of Objects'));
                 return;
             }
 
+            this.dropUnconfiguredIndices(procedures, function(err) {
+                if(util.isError(err)) {
+                    pb.log.error('DBManager: Error occurred during index check/deletion ERROR[%s]', err.stack);
+                }
```

Comment:
No I did not.  Now the error is passed into a callback with a return to end the function.

Ben Tidwell
Software Engineer III | Talent Network
CareerBuilder
5550-A Peachtree Pkwy, Norcross GA 30092

From: Brian <notifications@github.com<mailto:notifications@github.com>>
Reply-To: pencilblue/pencilblue <reply@reply.github.com<mailto:reply@reply.github.com>>
Date: Sunday, July 12, 2015 at 7:02 PM
To: pencilblue/pencilblue <pencilblue@noreply.github.com<mailto:pencilblue@noreply.github.com>>
Cc: Ben Tidwell <ben.tidwell@careerbuilder.com<mailto:ben.tidwell@careerbuilder.com>>
Subject: Re: [pencilblue] Multi Tenancy (#680)


In include/dao/db_manager.js<https://github.com/pencilblue/pencilblue/pull/680#discussion_r34426743>:

>              if (!util.isArray(procedures)) {
>                  cb(new Error('The procedures parameter must be an array of Objects'));
>                  return;
>              }
>
> +            this.dropUnconfiguredIndices(procedures, function(err) {
> +                if(util.isError(err)) {
> +                    pb.log.error('DBManager: Error occurred during index check/deletion ERROR[%s]', err.stack);
> +                }


did you mean to silently fail here?

-
Reply to this email directly or view it on GitHub<https://github.com/pencilblue/pencilblue/pull/680/files#r34426743>.